src/entities/state/state.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { State } from './state.entity';
+import { StateElectionResultService } from './../state-election-result/state-election-result.service';
+import { StateElectionResult } from '../state-election-result/state-election-result.entity';
+
+@Injectable()
+export class StateService {
+  constructor(
+    @InjectRepository(State)
+    private readonly stateRepository: Repository<State>,
+    private readonly stateElectionResultService: StateElectionResultService,
+  ) {}
+
+  findAll(): Promise<State[]> {
+    return this.stateRepository.find();
+  }
+
+  findOne(id: number): Promise<State> {
+    return this.stateRepository.findOneBy({ id });
+  }
+
+  findElectionResults(id: number): Promise<StateElectionResult[]> {
```

Comment:
Why do we need to duplicate this method in State if we have `state-election-entity` with results?

Reply:
Done. I've removed duplicated methods in services and injected them into controllers